pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -206,7 +210,20 @@ def __call__(self, value, system):
             for element in text_element:
                 restriction_on_landownership[element] = '\n'.join(restriction_on_landownership[element])
 
-        extract_dict['RealEstate_RestrictionOnLandownership'] = list(theme_restriction.values())
+        restrictions = list(theme_restriction.values())
+        for restriction in restrictions:
+            legends = {}
+            for legend in restriction['Legend']:
+                type_ = legend['TypeCode']
+                if type_ in legends:
+                    for item in ['Area', 'Length', 'PartInPercent']:
+                        if legend.get(item) is not None:
```

Comment:
should be either `type_` or `type` from L#217 down to L#223 in my opinion (and given my tests 😉)

and we have an other issue here. Somewhere on the way 'PartInPercent' is set to None instead of 0.0 when the area of a restriction is very small compared to the whole parcel: I have a case where a restriction area is 2.75m2 and the overall property is 158'596m2 >> None

So until we find where this assignement happens, I suggest this workaraound:
 ```
if legend.get(item) is not None:
    if legends[type_][item] is None:
        legends[type_][item] = 0.0
    legends[type_][item] += float(legend[item])
else:
    legends[type_] = legend
```

Reply:
that gives me this PDF: 
[20171024031649_extract.pdf](https://github.com/camptocamp/pyramid_oereb/files/1410932/20171024031649_extract.pdf)

Reply:
The "Distances légales à la forêt" don't have any values, does it have any value here?
https://github.com/camptocamp/pyramid_oereb/pull/287/files#diff-cfb7a95e0585c750deca6eabb5c5119bR167

Reply:
I try to have a better fix for the 0 values :-)